index.html

```diff
@@ -0,0 +1,389 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="queries.css" />
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+    <script
+      nomodule
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
+    <title>Online School</title>
+  </head>
+  <body>
+    <!-- main page header -->
+    <div class="container">
+      <header class="header sticky">
+        <nav class="navigation">
+          <ul class="top-nav">
+            <li class="top-nav--element">
+              <a href="#"
+                ><ion-icon class="facebook-icon" name="logo-facebook"></ion-icon
+              ></a>
+            </li>
+            <li class="top-nav--element">
+              <a href="#"
+                ><ion-icon class="twitter-icon" name="logo-twitter"></ion-icon
+              ></a>
+            </li>
+            <li class="top-nav--element"><a href="#">English</a></li>
+            <li class="top-nav--element"><a href="#">My Page</a></li>
+            <li class="top-nav--element"><a href="#">Logout</a></li>
+          </ul>
+          <a href="index.html">
+            <img
+              class="logo"
+              src="./img/nav-logo.jpg"
+              alt="website logo"
+              width="100px" />
+          </a>
+          <button class="btn-mobile-nav">
+            <ion-icon class="btn__nav btn__open" name="menu-outline"></ion-icon>
+            <ion-icon
+              class="btn__nav btn__close"
+              name="close-outline"></ion-icon>
+          </button>
+          <ul class="navigation__list">
+            <li class="navigation__list--links">
+              <a href="about.html" style="color: #e32816; border-color: #e32816"
+                >About</a
+              >
+            </li>
+            <li class="navigation__list--links"><a href="#">Home</a></li>
+            <li class="navigation__list--links"><a href="#">Join</a></li>
+            <li class="navigation__list--links"><a href="#">Sponsor</a></li>
+            <li class="navigation__list--links"><a href="#">News</a></li>
+            <button class="navtigation--btn">Campaign</button>
+          </ul>
+        </nav>
+      </header>
+
+      <!-- HOME Page Section -->
+      <section class="home__section">
+        <h1 class="home__primary--header">
+          <span>"hello! sharing world"</span>
+          <br>
+          experience 
+          a better eudcation<br /> with us
+        </h1>
+        <p class="home__description">
+          your childrens will have a better learing experience With Us
+        </p>
```

Comment:
- [ ] As requested by the previous reviewer please fix the typo error, like  Please write "Education" instead of "eudcation" and "Children" instead of "Children's"

![image](https://user-images.githubusercontent.com/75126481/217825747-bec82373-1382-4946-99f1-074b60eea22f.png)